terminus-ui/scss/helpers/_color.scss

```diff
@@ -226,7 +226,7 @@ $color__highlight: #efe557 !default;
  * @color
  * @section UI: Highlight
  */
-$color__highlight--light: #f4eb6e !default;
+$color__highlight--light: #f6ef8b !default;
```

Comment:
Does UX know (or need to know) about this color change?

Reply:
Nope. This color is currently not in use anywhere.